src/worker_utils.ts

```diff
@@ -0,0 +1,62 @@
+import { Pool, PoolClient } from "pg";
+import { WorkerUtilsOptions, TaskOptions, Job } from "./interfaces";
+import { makeWithPgClientFromPool, makeAddJob } from "./helpers";
+import { migrate } from "./migrate";
+import { defaultLogger } from "./logger";
+import { withReleasers, assertPool } from "./runner";
+
+const processPublisherOptions = async (options: WorkerUtilsOptions) => {
+  const { logger = defaultLogger } = options;
+  return withReleasers(async (releasers, _) => {
+    const pgPool: Pool = await assertPool(options, releasers, logger);
+
+    const withPgClient = makeWithPgClientFromPool(pgPool);
+
+    // Migrate
+    await withPgClient(client => migrate(client));
```

Comment:
I think we should move this to a `migrate` method; calling this every time you call `require('graphile-worker').addJob(...)` could get expensive.

Reply:
Added a migrate method.